samples/msal-browser-samples/VanillaJSTestApp2.0/cert.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Shouldn't this be read from a keyvault?

Reply:
Removed and added npm script to generate new certs on demand